qmd/tables.qmd

```diff
@@ -63,15 +63,18 @@ source("R/qmd-utils.R")
 source("R/format-table.R")
 # Source warning utils
 source("R/warning-utils.R")
-source("R/dictionary-module.R")
-source("R/parts-module.R")
-
+source("R/odm-dictionary-file.R")
+source("R/parts-sheet.R")
+# String set Values
+parts$part_sheet_table_column_type_set_PK <- "Primary Key"
```

Comment:
Why are you adding these to the `parts` object rather than as independent variables?